packages/api/src/sdk-bundle-api/bundle-to-sdk/screenshot-diff-result.ts

```diff
@@ -81,6 +80,18 @@ export interface ScreenshotDiffResultCompared {
   mismatchFraction: number;
 }
 
+export interface ScreenshotDiffResultNoDifference
+  extends ScreenshotDiffResultCompared {
+  outcome: "no-diff";
+}
+
+export interface ScreenshotDiffResultDifference
+  extends ScreenshotDiffResultCompared {
+  outcome: "diff";
+
+  hashOfChangedSectionsClassNames?: string;
```

Comment:
[nit / not important / can skip] Could always add a comment here to explain what this is / how it's computed